BMM.Core/ViewModels/Interfaces/IShareTrackCollectionViewModel.cs

```diff
@@ -0,0 +1,11 @@
+using BMM.Core.ViewModels.Parameters.Interface;
+using MvvmCross.ViewModels;
+
+namespace BMM.Core.ViewModels.Interfaces
+{
+    public interface IShareTrackCollectionViewModel : IMvxViewModel<ITrackCollectionParameter>
```

Comment:
Not sure about that interface. Could you shortly explain it to me?

Reply:
We should start adding interfaces to VMs if we want to use Guarded Actions in the future.